familyplanningmodule/src/main/res/drawable/rounded_corner_edit_text.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="rectangle">
+            <solid android:color="#1E81BD" />
+            <corners
+                android:bottomRightRadius="4dp"
+                android:bottomLeftRadius="4dp"
+                android:topRightRadius="4dp"
+                android:topLeftRadius="4dp" />
+        </shape>
+    </item>
+
+    <item
+        android:bottom="0.5dp"
+        android:top="0.5dp"
+        android:left="0.5dp"
+        android:right="0.5dp">
+        <shape android:shape="rectangle">
+            <solid android:color="#fafafa" />
```

Comment:
Can you move these text-colors and text-color-hints to the `colors.xml` files and reuse them